spec/controllers/votes_controller_spec.rb

```diff
@@ -5,17 +5,27 @@
   include TestFactories
   include Devise::TestHelpers
 
+  before do
+    request.env["HTTP_REFERER"] = '/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.